views/forms/event/add.php

```diff
@@ -40,7 +40,10 @@
 <label><?php esc_html_e( 'Arguments', 'advanced-cron-manager' ); ?></label>
 <p class="description"><?php esc_html_e( 'New inputs will be added automatically when you type', 'advanced-cron-manager' ); ?></p>
 <div class="event-arguments">
-	<input type="text" name="arguments[]" class="event-argument widefat">
+	<div class="event-argument-wrapper">
```

Comment:
Why another div here? Can't you use the `.event-arguments`?

Reply:
we need a container for all wrappers, so we can attach jquery to it and put event-argument-wrappers inside it.
<img width="365" alt="Screenshot 2022-11-08 at 17 57 47" src="https://user-images.githubusercontent.com/70862071/200627841-2f0cc0d2-9bed-4913-91f4-856f9c261eff.png">

Reply:
@kryswoj just to make sure: I'd like the buttons to be always below the inputs and the `X` icon beside the input, not below. If wrapper is needed to get that, it's fine